games-puzzle/lpairs/lpairs-1.0.4.recipe

```diff
@@ -27,15 +27,18 @@ BUILD_REQUIRES="
 	"
 BUILD_PREREQUIRES="
 	cmd:make
-	cmd:gcc
+	cmd:g++
 	cmd:ld
 	cmd:libtoolize
 	"
+if [ "$effectiveTargetArchitecture" == "x86_64" ] ; then
+PATCHES="lpairs-1.0.4.patchset"
```

Comment:
**WARNING:** All of this is just speculation, no actual testing has been done

It appears to me that your patchset won't cause trouble for gcc2, so you could safely remove this conditional